react-app/ccf/src/pages/applicant_results/ResultsPage.css

```
@@ -0,0 +1,136 @@
+.reviews h2{
+    font-size: 0.85rem;
+}
+
+.review-box{
+    width: 100%;
+    padding-right: 15px;
+    margin-top: 5px;
+    margin-bottom: 1rem;           
+    border-width: 5px;
+    background: #e3e3e3;          
+    border: 1px solid #575757;    
```

Biome (2.1.2)

[error] Unexpected shorthand property border after border-width



(lint/suspicious/noShorthandPropertyOverrides)